acceptance/acceptance_suite_test.go

```diff
@@ -284,6 +294,13 @@ func uninstallEpinio() (string, error) {
 	return Epinio("uninstall", "")
 }
 
+// Remove all tmp directories from /tmp/epinio-* . Test should try to cleanup
+// after themselves but that sometimes doesn't happen, either because we forgot
+// the cleanup code or because the test failed before that happened.
+func cleanupTmp() (string, error) {
+	return RunProc("rm -rf /tmp/epinio-*", "", false)
```

Comment:
This would be a problem is someone was running 2 instances of the acceptance test suite in parallel on the same machine. As soon as the first instance would be done, it would delete all directories of  both instances. I don't think that is very likely to happen but it's good to keep in mind.